src/Notmuch.hs

```diff
@@ -72,6 +72,7 @@ module Notmuch
   , queryCountThreads
 
   , Thread
+  , thread_top_messages
```

Comment:
The main API should use camelCase. I think it should just be called `topLevelMessages`.

Reply:
Will change!